docs/source/architecture/rest_api.rst

```diff
@@ -0,0 +1,293 @@
+********
+REST API
+********
+
+*Hyperledger Sawtooth* provides a pragmatic RESTish API for clients to interact
+with a validator using common HTTP/JSON standards. It is an entirely separate
+process, which once running allows transactions to be submitted and blocks to
+be read with a common language-neutral interface. As the validator is redesigned
+and improved the REST API will grow with it, providing a consistent interface
+that meets the needs of app developers into the future.
+
+With that focus on app developers, the REST API treats the validator mostly as a
+black box, submitting transactions and fetching the results. It is not a tool
+for *all* validator communication. It would not be used by Transaction
```

Comment:
Fair point - we should probably remove the ledger sync example, as we replaced it with state export.